pages/coinbase/homepage.py

```diff
@@ -1,45 +1,237 @@
+    bitcoin_BTC_link_loc =          (by.XPATH, "//h4[.='BTC']")
+    bitcoin_logo_link_loc =         (by.XPATH, "//img[@alt='Bitcoin logo']")
+    bitcoin_price_button_loc =      (by.XPATH, "//a[@href='/price/bitcoin']/button")
+
+    ethereum_text_link_loc =        (by.XPATH, "//h4[.='Ethereum']")
+    ethereum_ETH_link_loc =         (by.XPATH, "//h4[.='ETH']") 
+    ethereum_logo_link_loc =        (by.XPATH, "//img[@alt='Ethereum logo']")
+    ethereum_price_button_loc =     (by.XPATH, "//a[@href='/price/ethereum']/button")
+
+    bitcoinCash_text_link_loc =     (by.XPATH, "//h4[.='Bitcoin Cash']")        
+    bitcoinCash_BCH_link_loc =      (by.XPATH, "//h4[.='BCH']")
+    bitcoinCash_logo_link_loc =     (by.XPATH, "//img[@alt='Bitcoin Cash logo']")
+    bitcoinCash_price_button_loc =  (by.XPATH, "//a[@href='/price/bitcoin-cash']/button")
+
+    litecoin_text_link_loc =        (by.XPATH, "//h4[.='Litecoin']")
+    litecoin_LTC_link_loc =         (by.XPATH, "//h4[.='LTC']")
+    litecoin_logo_link_loc =        (by.XPATH, "//img[@alt='Litecoin logo']")
+    litecoin_price_button_loc =     (by.XPATH, "//a[@href='/price/litecoin']/button")
+
+    # EARNING LINKS BLCOK
+
+    start_earning_button_loc =       (by.XPATH, "//button[contains(@class,'EarnUpsellBanner')]")
+    start_earning_lower_button_loc = (by.XPATH, "//button[contains(@class,'EarnFooterBanner')]")
+    maker_link_loc =                 (by.XPATH, "//a[@href='/earn/maker']/div")
+    celo_link_loc =                  (by.XPATH, "//a[@href='/earn/celo']/div/div/h2")   #I don't love this xpath, but I feel it is the best way to locate this element
+    compound_link_loc =              (by.XPATH, "//a[@href='/earn/compound']/div/div/h2")   #ditto
+    EOS_link_loc =                   (by.XPATH, "//a[@href='/earn/eos']/div/div/h2")
+    view_more_link_loc =             (by.XPATH, "//a[@href='/earn']/div")
+
+    # MOBIL APPS
+    
+    android_app_link_loc =           (by.XPATH, "//a[contains(@href,'android')]")#='Android']")
+    ios_app_link_loc =               (by.XPATH, "//a[.='iOS']")
+
+    # COINBASE SECURITY/INSURANCE
+
+    security_link_loc =             (by.XPATH,"//a[contains(@title,'funds') or contains(@title,'secure')]") #either funds or secure works' alone, I feel doing this increases longengevity
+    insurance_link_loc =            (by.XPATH,"//a[contains(@title,'insurance') or contains(@title,'policy')]")     #same as above
+    best_practices_link_loc =       (by.XPATH,"//a[contains(@title,'industry') or contains(@title,'practices')]")   #at first I thought I'd use an and instead of an or in case another title is created with just one of these words in it.
+
+    # LANGUAGE MENU FOOTER
+
+    language_menu_loc =             (by.XPATH,"//select[contains(@class,'Language')]")
+
+    # PRODUCTS FOOTER
+
+    coinbase_link_loc =             (by.XPATH, "//a[.='Coinbase']")     #from here down I feel like maybe this is an easy/lazy way to finish the homepage
```

Comment:
What does this comment mean?

Reply:
I just felt that xpath was too easy, maybe I was being lazy. The rest of my xpaths are like this